cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -101,8 +101,8 @@
 		@Tag(name = "Measure Evaluation", description = "IBM Cohort Engine Measure Evaluation"),
 		@Tag(name = "ValueSet", description = "IBM Cohort Engine ValueSet Operations")})
 public class CohortEngineRestHandler {
-	private static final String EVALUATION_API_NOTES = "The body of the request is a multipart/form-data request with an application/json attachment named 'request_data' that describes the measure evaluation that will be performed and an application/zip attachment named 'measure' that contains the measure and library artifacts to be evaluated. Valueset resources required for Measure evaluation must be loaded to the FHIR server in advance of an evaluation request. ";
-	private static final String COHORT_EVALUATION_API_NOTES = "The body of the request is a multipart/form-data request with  an application/zip attachment named 'cql_definition' that contains the cohort cql definition to be evaluated.";
+	private static final String EVALUATION_API_NOTES = "The body of the request is a multipart/form-data request with an application/json attachment named 'request_data' that describes the measure evaluation that will be performed and an application/zip attachment named 'measure' that contains the measure and library artifacts to be evaluated. Valueset resources required for Measure evaluation must be loaded to the FHIR server in advance of an evaluation request. Examples of the response measure reports (individual and patient list) can be found as part the FHIR IG: https://www.hl7.org/fhir/measurereport-examples.html.";
+	private static final String COHORT_EVALUATION_API_NOTES = "The body of the request is a multipart/form-data request with  an application/zip attachment named 'cql_definition' that contains the cohort cql definition to be evaluated. The response object is a json payload with the list of passing patients: {\"result\": [\"patient1\", \"patient2\"] }";
```

Comment:
Instead of documenting it here, the correct way to document the return data, is to define an object to contain the response data and then reference it in the @ApiResponse (see @ApiResponse(code = 200, message = "Successful Operation", response = MeasureParameterInfoList.class),). This way, the object will get rendered in the Responses section of the swagger.

The current way is hacking together a string representation on line 432 instead of using an object like we really should be.